tests-legacy/Services/EmployeeLoginTrait.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace LegacyTests\Services;
+
+use Context;
+use PrestaShop\PrestaShop\Core\Domain\Employee\AuthorizationOptions;
+use PrestaShopBundle\Security\Admin\Employee as LoggedEmployee;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorage;
+use Symfony\Component\Security\Core\Authentication\Token\UsernamePasswordToken;
+use Symfony\Component\Security\Core\Role\Role;
+
+trait EmployeeLoginTrait
+{
+    /**
+     * Emulates a real employee logged to the Back Office.
+     * Must be used inside a TestCase
+     */
+    protected function logIn()
```

Comment:
What's the use of this method? Since there is already an `employeeLogin` and a `symfonyLogin` methods?